packages/@azure/servicebus/data-plane/test/namespace.spec.ts

```diff
@@ -8,6 +8,9 @@ import dotenv from "dotenv";
 dotenv.config();
 chai.use(chaiAsPromised);
 import { Namespace, delay } from "../lib";
+import * as msrestAzure from "ms-rest-azure";
+import { getSenderReceiverClients, ClientType, testSimpleMessages, getEnvVars } from "./testUtils";
+const aadServiceBusAudience = "https://servicebus.azure.net/";
```

Comment:
Can you create a PR for amqp-common for this?

Reply:
https://github.com/Azure/amqp-common-js/issues/30